.release

```diff
@@ -1 +0,0 @@
-23.11
```

Comment:
Not sure we can remove this outright as people may be relying on it existing. Probably safest to leave it as-is and make a release note saying that the file is deprecated and will be removed in the 24.05 release cycle.

Reply:
Good point; done. I was thinking of this as more of an implementation detail than an external API, but you can never know what people are using.